viewer/packages/pointcloud-styling/src/utils.ts

```diff
@@ -0,0 +1,18 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+
+import { DMInstanceRefPointCloudObjectCollection } from './DMInstanceRefPointCloudObjectCollection';
+import { PointCloudObjectCollection } from './PointCloudObjectCollection';
+
+export function isPointCloudObjectCollection(
+  collection: PointCloudObjectCollection | DMInstanceRefPointCloudObjectCollection
```

Comment:
I wonder if we should deprecate `PointCloudObjectCollection` and find a new name to make it clear that it's based on the earlier annotations? E.g. `PointCloudAnnotationVolumeCollection`? And use the `VolumeCollection` in the new class as well? What do you think? (Then we would need to remember to update the name of this function too :sweat_smile: